dev/PushNotifications/PushNotificationsLongRunningTask.StartupTask/PushNotificationsLongRunningPlatform-Startup.h

```diff
@@ -5,6 +5,8 @@
 
 #include <wil/com.h>
 #include <wil/resource.h>
+#include "../../Common/Microsoft.RoApi.h"
+#include "../../PushNotifications/NotificationPlatformActivation.h"
```

Comment:
In general, we should avoid relative paths for header files. Instead the project file should include the dirs that we care about and then, this wouldn't be needed.

Same comment for other files where relative headers are used.